Scripts/script-CSVHunter.yml

```diff
@@ -0,0 +1,172 @@
+commonfields:
+  id: CSVHunter
+  version: -1
+name: CSVHunter
+script: |-
+  import re
+  import csv
+  import socket
+
+  def is_valid_ipv4_address(address):
+      try:
+          socket.inet_pton(socket.AF_INET, address)
+      except AttributeError:  # no inet_pton here, sorry
+          try:
+              socket.inet_aton(address)
+          except socket.error:
+              return False
+          return address.count('.') == 3
+      except socket.error:  # not a valid address
+          return False
+      return True
+
+  csvEntry = None
+
+  iplist = []
+  domainlist = []
+  hashlist = []
+  md5hashlist = []
+  sha1hashlist = []
+  sha256hashlist = []
+  ipcount = 0
+  domaincount = 0
+  hashcount = 0
+  md5hashcount = 0
+  sha1hashcount = 0
+  sha256hashcount = 0
+
+  dArgs = demisto.args()
+
+  file = dArgs['file'] if 'file' in dArgs else None
+
+
+  entries = demisto.executeCommand('getEntries', {})
+  for entry in entries:
+      fn = demisto.get(entry, 'File')
+
+      is_text = type(fn) in [unicode, str]
```

Comment:
Not sure I understand what this line does - check that the file name is string? Why?